src/unix/core.c

```diff
@@ -905,9 +905,11 @@ static int maybe_resize(uv_loop_t* loop, unsigned int len) {
 
 void uv__io_cb(uv_loop_t* loop, uv__io_t* w, unsigned int events) {
   switch (uv__io_cb_get(w)) {
+  #if ! defined(__PASE__)
   case UV__AHAFS_EVENT:
     uv__ahafs_event(loop, w, events);
     break;
+  #endif
```

Comment:
Don't do that in core.c, define it as a stub function in aix.c if HAVE_SYS_AHAFS_EVPRODS_H (I think?) is not defined.

Reply:
I opened #4903 just now that does that.